addons/mail/static/src/rtc/call_action_list.xml

```diff
@@ -62,43 +51,25 @@
                             <i class="fa fa-video-camera fa-stack-1x" t-att-class="{ 'fa-lg': !isSmall, 'text-success': rtc.state.sendCamera }"/>
                         </div>
                     </button>
-                    <t t-if="rtc.state.sendScreen" t-set="screenSharingText">Stop screen sharing</t>
-                    <t t-else="" t-set="screenSharingText">Share screen</t>
-                    <button t-if="!isMobileOS" class="btn d-flex border-0 rounded-circle shadow-none opacity-100 opacity-75-hover"
-                        t-att-class="{
-                            'p-2': isSmall,
-                            'p-3 m-1': !isSmall,
-                        }"
-                        t-att-aria-label="screenSharingText"
-                        t-att-title="screenSharingText"
-                        t-on-click="() => this.rtc.toggleVideo('screen')">
-                        <div class="fa-stack">
-                            <i class="fa fa-desktop fa-stack-1x" t-att-class="{ 'fa-lg': !isSmall, 'text-success': rtc.state.sendScreen }"/>
-                        </div>
-                    </button>
-                    <button t-if="!props.fullscreen.isActive" class="btn d-flex border-0 rounded-circle shadow-none opacity-100 opacity-75-hover"
-                        aria-label="Enter Full Screen"
-                        title="Enter Full Screen"
-                        t-att-class="{ 'p-2': isSmall, 'p-3 m-1': !isSmall }"
-                        t-on-click="props.fullscreen.enter"
-                    >
-                        <div class="fa-stack">
-                            <i class="fa fa-arrows-alt fa-stack-1x" t-att-class="{ 'fa-lg': !isSmall }"/>
-                        </div>
-                    </button>
-                    <button t-if="props.fullscreen.isActive" class="btn d-flex border-0 rounded-circle shadow-none opacity-100 opacity-75-hover"
-                        aria-label="Exit Full Screen"
-                        title="Exit Full Screen"
-                        t-att-class="{ 'p-2': isSmall, 'p-3 m-1': !isSmall }"
-                        t-on-click="props.fullscreen.exit"
-                    >
-                        <div class="fa-stack">
-                            <i class="fa fa-compress fa-stack-1x" t-att-class="{ 'fa-lg': !isSmall }"/>
-                        </div>
-                    </button>
+                    <Dropdown position="'top-end'" togglerClass="`btn d-flex m-1 border-0 rounded-circle
+                    shadow-none opacity-100 opacity-75-hover ${ isSmall ? 'p-2' : 'p-3' }`" menuClass="'d-flex flex-column py-0'">
+                        <t t-set-slot="toggler">
+                            <div class="fa-stack">
+                                <i class="fa fa-ellipsis-v fa-stack-1x" t-att-class="{ 'fa-lg': !isSmall }" title="More" aria-label="More"/>
```

Comment:
`title` and `aria-label` are useless in `<i>` of "More" button, as the `Dropdown` has already `title` with exact same value